fingerprintdb_cuda.cu

```diff
@@ -233,7 +337,13 @@ float FingerprintDB::tanimoto_similarity_cpu(const Fingerprint& fp1,
 size_t get_available_gpu_memory()
```

Comment:
I assume theres no other gpu stuff in rdkit and a need to abstract this? It'd be nice if cuda just had a function that did this query for you...

Reply:
Don't think so